package.json

```diff
@@ -42,6 +42,7 @@
   ],
   "devDependencies": {
     "@lhci/cli": "^0.10.0",
+    "@storybook/html": "*",
```

Comment:
Actually can you make this a peer depencency in this file? The goal is to force npm to resolve just a single version of the package (making it a dev dep could allow multiple nested versions)